app.js

```diff
@@ -1,14 +1,16 @@
-const express = require('express')
-const app = express()
-const port = 3000
-
-app.get('/', (req, res) => {
-  res.send('Hello World!')
-})
-app.get('/health', (req, res) => {
-  res.send('Health Checked!');
+const express = require('express');
+const base64url = require('base64url');
+
+const app = express();
+const port = 3000;
+
+app.get('/text-base64', (req, res) => {
+  const { text } = req.query;
+  const base64 = base64url.encode(text);
+
+  res.json({ text, base64 });
```

Comment:
This api should return the below response body when we send `hello` as text input.
```
{
    "text": "hello",
    "base64": "aGVsbG8="
}
```